week-1/exercises/initials.js

```diff
@@ -2,5 +2,26 @@ let firstName = "Creola";
 let middleName = "Katherine";
 let lastName = "Johnson";
 
+// create the initials together
+let initials =
+  "First Name Initials: " +
+  firstName.slice(0, 3).toUpperCase() +
+  "\n" +
+  "Middle Name Initials: " +
+  middleName.slice(0, 3).toUpperCase() +
+  "\n" +
+  "Last Name Initials: " +
+  lastName.slice(0, 3).toUpperCase();
+
+// note to myself
+/* 
+another way 
+let initials = `First Name Initials: ${firstName.slice(0, 3).toUpperCase()}\n
+                Middle Name Initials: ${middleName.slice(0, 3).toUpperCase()}\n
+                Last Name Initials: ${lastName.slice(0, 3).toUpperCase()}`;
```

Comment:
I'm not sure what this exercise is actually supposed to do, but your code is fine so 👍